### PR TITLE
(DOCSP-21147): Refactor to improve readability to B grade or grade 9

### PR DESCRIPTION
**Pull request by dacharyc**
### Jira

- https://jira.mongodb.org/browse/DOCSP-21147

### Staged Changes (Requires MongoDB Corp SSO)

- [Document Preimages](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-21147/mongodb/preimages/): Refactor to improve readability from D to B in Readability, or grade 11 to grade 9 in Hemingway. Screenshots in attached Jira ticket. This also updates the screenshot in the Create Database Triggers page, which was missing the UI element to enable document preimages.

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/mongodb/preimages.txt [hunk 1]
```diff
@@ -38,7 +36,7 @@ write throughput.
          { $inc: { age: 1 } }
       )
 
-   The change event for the operation includes:
+   Preimages add this data to each change event for write operations:
```
Comment (MongoCaleb) [inline source/mongodb/preimages.txt:39]: "this data" or "the following data"?

File: source/mongodb/preimages.txt [hunk 2]
```diff
@@ -65,43 +63,45 @@ write throughput.
 Enable Document Preimages
 -------------------------
 
-Your app automatically enables document preimages for a collection when
-you define a trigger on that collection that uses preimages. Once you
-have enabled preimages for a collection, you can configure any number of
-triggers on the collection to include a document preimage in the change
-event.
+You can enable preimages when you configure a database trigger. Document
```
Comment (MongoCaleb) [inline source/mongodb/preimages.txt:66]: Consider making "configure a database trigger" a link, and then remove the separate sentence below.

File: source/mongodb/preimages.txt [hunk 3]
```diff
@@ -65,43 +63,45 @@ write throughput.
 Enable Document Preimages
 -------------------------
 
-Your app automatically enables document preimages for a collection when
-you define a trigger on that collection that uses preimages. Once you
-have enabled preimages for a collection, you can configure any number of
-triggers on the collection to include a document preimage in the change
-event.
+You can enable preimages when you configure a database trigger. Document 
+Preimage is a setting on trigger configuration. Toggle this to enable
```
Comment (MongoCaleb) [inline source/mongodb/preimages.txt:67]: "Document Preimage" without any formatting is hard to parse. Consider bolding or UIing it...or rewriting:

You can enable preimages when you <link>configure a database trigger</link> by toggling the Document Preimage option.

File: source/mongodb/preimages.txt [hunk 4]
```diff
@@ -65,43 +63,45 @@ write throughput.
...
+2. Expand the :guilabel:`Advanced Configuration` section.
+
+This section contains the :guilabel:`Preimage Preferences by Collection` 
+table.
 
-The table lists every collection that stores preimages in the oplog.
-This includes all collections in the linked cluster, even if no triggers
-currently use the preimages for a collection or if the triggers are in a
-different Realm app.
+This table lists every collection that stores preimages in the oplog.
```
Comment (MongoCaleb) [inline source/mongodb/preimages.txt:86]: do we need to link to the oplog definition?

File: source/mongodb/preimages.txt [hunk 5]
```diff
@@ -65,43 +63,45 @@ write throughput.
...
+
+This section contains the :guilabel:`Preimage Preferences by Collection` 
+table.
 
-The table lists every collection that stores preimages in the oplog.
-This includes all collections in the linked cluster, even if no triggers
-currently use the preimages for a collection or if the triggers are in a
-different Realm app.
+This table lists every collection that stores preimages in the oplog. 
+Enabling preimages for a collection applies to all collections. This
```
Comment (MongoCaleb) [inline source/mongodb/preimages.txt:87]: Huh? Enabling it for one collection enables it for all collections within a database? If so, wouldn't the table simply show all collections?

File: source/mongodb/preimages.txt [hunk 6]
```diff
@@ -65,43 +63,45 @@ write throughput.
...
+You can disable collection-level preimages in the Realm UI. To disable 
 preimages for a collection:
 
-1. (Optional) Disable document preimages for all triggers on the collection.
-   When collection-level preimages are disabled, triggers on the collection that
-   still have preimages enabled (including triggers in other Realm apps) will
-   continue to execute but will not include the ``fullDocumentBeforeChange``
-   field in change events.
+1. (Optional) Disable document preimages for triggers in the collection.
+   Triggers with preimages work after disabling preimages. But they don't
```
Comment (MongoCaleb) [inline source/mongodb/preimages.txt:98]: > You can enable preimages when you configure a database trigger 

...needs further explanation. How can this be possible?

Comment (MongoCaleb) [inline source/mongodb/preimages.txt:98]: I know that various text analyzers prefer shorter sentences, but starting a sentence with "But" or "And" is a major writing flag for me. I'm sure it can be argued; personally, I *hate* it. :shrug:

File: source/mongodb/preimages.txt [hunk 7]
```diff
@@ -65,43 +63,45 @@ write throughput.
...
 
 2. Disable sync, if the collection is part of a :ref:`synced cluster <sync>`.
 
-3. On the linked cluster configuration screen, find the collection in the
-   :guilabel:`Preimage Preferences by Collection` table and then click
-   :guilabel:`Disable`.
+3. View :ref:`the Preimage Preferences By Collection 
+   <view-preimage-enabled-collections>` table. Press the
+   :guilabel:`Disable` button for a collection. This turns off preimages for 
+   that collection.
```
Comment (MongoCaleb) [inline source/mongodb/preimages.txt:107]: Do we need a step 4 to re-enable Sync?

File: source/mongodb/preimages.txt (hunk 3 above)
Comment (dacharyc) [reply source/mongodb/preimages.txt:67]: Ahh, yeah, sorry, this should have been a :guilabel: - missed that.

File: source/mongodb/preimages.txt (hunk 4 above)
Comment (dacharyc) [reply source/mongodb/preimages.txt:86]: We link to oplog in ln 24 above. Do we normally link out for a term more than once on a page?

File: source/mongodb/preimages.txt (hunk 5 above)
Comment (dacharyc) [reply source/mongodb/preimages.txt:87]: That was my reading from the original page text: [View Preimage-Enabled Collections](https://www.mongodb.com/docs/realm/mongodb/preimages/#view-preimage-enabled-collections). 

> The table lists every collection that stores preimages in the oplog. This includes all collections in the linked cluster, even if no triggers currently use the preimages for a collection or if the triggers are in a different Realm app.

I thought that was a bit strange so I experimented with this feature and all collections did show up in this table.

File: source/mongodb/preimages.txt (hunk 6 above)
Comment (dacharyc) [reply source/mongodb/preimages.txt:98]: Happy to clarify further on the page, but I'm not sure what you're asking here. Can you elaborate, please?

Comment (dacharyc) [reply source/mongodb/preimages.txt:98]: As long as the sentence is not a sentence fragment, there's no grammatical rule that says you can't start a sentence with a conjunction. Many of us were taught this in school, but it isn't actually "against the rules." It _can_ be against convention, if we're following a style guide that prohibits it.

I consulted our style guide and couldn't find any specific guidance around this. It does say we follow the _Chicago Manual of Style_, so I checked there for more clarification and found this FAQ item: [Is it proper (or good) academic form to begin a sentence with a conjunction](https://www.chicagomanualofstyle.org/qanda/data/faq/topics/Usage/faq0013.html)

File: source/mongodb/preimages.txt (hunk 7 above)
Comment (dacharyc) [reply source/mongodb/preimages.txt:107]: Oof, yeah, you're right, good catch. Which means we also need the warning about terminating and re-enabling Sync that I recently added to a bunch of pages. Adding it here!

File: source/mongodb/preimages.txt (hunk 6 above)
Comment (MongoCaleb) [reply source/mongodb/preimages.txt:96]: Sorry -- what I wondered is how can triggers with preimages still work if I disable premimages?  That's really counterintuitive.

Comment (MongoCaleb) [reply source/mongodb/preimages.txt:96]: Good link & counterpoint. I still hold, as CMOS says, "...only when the result is perfectly clear and more effective than some other alternative." :)

File: source/mongodb/preimages.txt (hunk 1 above)
Comment (MongoCaleb) [reply source/mongodb/preimages.txt:39]: (I prefer the latter as it is more clear which data you are referring to)

**Review by MongoCaleb — APPROVED** [review]
Replied to 3 comments, which I trust you to implement or ignore as you see fit. :)

File: source/mongodb/preimages.txt (hunk 6 above)
Comment (dacharyc) [reply source/mongodb/preimages.txt:96]: Ahh, gotcha! Rewording this paragraph to try to make it clearer.
> (Optional) Disable document preimages for triggers in the collection.
   Triggers with preimages continue to fire after disabling preimages. 
   But they don't have the ``fullDocumentBeforeChange`` field in change 
   events. This includes triggers in other Realm apps.

File: source/mongodb/preimages.txt (hunk 1 above)
Comment (dacharyc) [reply source/mongodb/preimages.txt]: "The following data" makes my inner editor twitch when this statement precedes a code block or example - especially when it's in a statement that ends with a colon - because the context is clear that the verbiage refers to the block that follows. The same editor who drilled this into me in my early writing years also dinged me for any instance of "This article" or "This page" for similar reasons, which we're also guilty of all over the docs.

Adding it back here because you seem to feel strongly about it, but it's definitely something I'd like to see us remove from the docs more broadly.

Comment (MongoCaleb) [reply source/mongodb/preimages.txt]: Fight Fight Fight! ;-)

File: source/mongodb/preimages.txt (hunk 6 above)
Comment (MongoCaleb) [reply source/mongodb/preimages.txt]: LGTM, except it _should_ be ", but" ;-P
